torchci/components/benchmark/llms/common.tsx

```diff
@@ -27,12 +27,22 @@ export const IS_INCREASING_METRIC_VALUE_GOOD: { [k: string]: boolean } = {
   flops_utilization: true,
   "compilation_time(s)": false,
   speedup: true,
+  "avg_inference_latency(ms)": false,
```

Comment:
I forgot that we need to update this to highlight the metrics correctly.  A better solution is probably need as part of https://github.com/pytorch/executorch/issues/8239